openapi/kas-fleet-manager.yaml

```diff
@@ -1016,6 +1016,15 @@ components:
               type: string
             max_connection_attempts_per_sec:
               type: integer
+            max_message_size:
+              description: "Maximum message size"
+              type: string
+            min_in_sync_replicas:
+              description: "Minimum number of in-sync replicas"
+              type: integer
+            replication_factor:
```

Comment:
can we add these new fields to the [KafkaRequestExample](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/dd5c689c8372d5e155cefcfeb47b53e7477d8028/openapi/kas-fleet-manager.yaml#L1656)?

Reply:
thanks!